pages/irac-2022.md

```diff
@@ -49,13 +47,79 @@ We have invited renowned speakers who will give an introductory overview for eac
+14:30-14:45 | **Gustavo Perez** - Using Spatio-Temporal Information in Weather Radar Data to Detect Communal Bird Roosts
+14:45-15:15 | Coffee break
+15:15-15:30 | **Yuting Deng** - Quantifying phenological trends in aerial insectivore roosting behaviors in the Great Lakes region using weather surveillance radar
+15:30-15:45 | **Fengyi Guo** - Autumn stopover hotspots and multi-scale habitat associations of migratory landbirds in the eastern U.S. **remote**{: .badge .bg-secondary }
+15:45-16:00	| **Benjamin Van Doren** - Linking acoustic monitoring and radar to capture the timing and intensity of bird migration **remote**{: .badge .bg-secondary }
+16:00-16:30	| Coffee break
+16:30-17:15	| **Jennifer Krauel** - Patterns in High-altitude Bat Movement over Texas Revealed by Radar **speed talk**{: .badge .bg-secondary } **remote**{: .badge .bg-secondary }
+&nbsp;      | **Virginia Halterman** - Patterns of flight altitudes for arriving spring migrants in the Gulf of Mexico region **speed talk**{: .badge .bg-secondary } **remote**{: .badge .bg-secondary }
+&nbsp;      | **Simon Hirschhofer** - High concentrations of migratory birds in Swiss-alpine valleys more frequent in pre- than postbreeding migration **speed talk**{: .badge .bg-secondary }
+&nbsp;      | **Thibault Désert** - SEMAFOR project: remote sensing of avifauna using the French meteorological radar network **speed talk**{: .badge .bg-secondary }
+&nbsp;      | **Jens van Erp** - Identifying fine-scale flight behaviour in 2D bird radar tracks: marine thermal soaring on the North Sea **speed talk**{: .badge .bg-secondary }
+&nbsp;      | **Abel Gyimesi** - Avian avoidance of offshore wind turbines measured by radar **speed talk**{: .badge .bg-secondary }
+17:15-20:00 | Drinks & poster session
+
+### Saturday - June 26
+
+time        | presentation
+----------- | ------------
+**Session 2**{: .text-primary } | **Methodological advancements - classification and identification algorithms, radar technology, data infrastructure**{: .text-primary }
+ 9:00-9:30  | **Buler Jeff** - Applied radar aeroecology
+ 9:30-9:45	| **Bart Hoekstra** - Fireworks disturbance across bird communities
+09:45-10:00 | **William Kunin** - Tracking desert locust swarms using dual-polarisation weather radar
+10:00-10:15 | **Zhenhua Hao** - Different migratory strategies in locusts and moths revealed by radar **remote**{: .badge .bg-secondary }
+10:15-10:30 | **Xu Shi** - Prospects for monitoring bird migration along the East Asian-Australasian Flyway using weather radar **speed talk**{: .badge .bg-secondary } **remote**{: .badge .bg-secondary }
+&nbsp;      | **Lili Jia** - Using radar to mitigate birdstrike risk in coastal airport on Bohai Bay **speed talk**{: .badge .bg-secondary } **remote**{: .badge .bg-secondary }		
+10:30-11:00 | Coffee break
+**Session 4**{: .text-primary } | **From radar aeroecology to biodiversity**{: .text-primary }
```

Comment:
@peterdesmet: 
```
If the session numbers refer to the [topics numbers](https://globam.science/irac-2022/#thematic-sessions), 
we should switch 3 and 4, because currently 4 is happening before 3. Let me know if you agree.
```
Simply switch the numbers of 4 and 3, yes. They indeed refer to the previous topic numbers. It doesn't matter that we switch them now.